scripts/dts/gen_defines.py

```diff
@@ -105,6 +106,103 @@ def parse_args():
     return parser.parse_args()
 
 
+def augment_node(node):
+    # Augment an EDT node with these zephyr-specific attributes, which
+    # are used to generate macros from it:
+    #
+    # - z_primary_ident: a primary node identifier based on the node's
+    #   compatible, plus information from its unit address (or its
+    #   parent's unit address) or its name, and/or its bus.
+    #
+    # - z_other_idents: a list of other identifiers for the node,
+    #   besides z_primary_ident.
+    #
+    # The z_other_idents list contains the following other attributes,
+    # concatenated in this order:
+    #
+    # - z_inst_idents: node identifiers based on the index of the node
+    #   within the EDT list of nodes for each compatible, e.g.:
+    #   ["INST_3_<NODE's_COMPAT>",
+    #    "INST_2_<NODE's_OTHER_COMPAT>"]
+    #
+    # - z_alias_idents: node identifiers based on any /aliases pointing to
+    #   the node in the devicetree source, e.g.:
+    #   ["DT_ALIAS_<NODE's_ALIAS_NAME>"]
```

Comment:
Missing `["DT_<NODE's_COMPAT>_<NODE's_ALIAS_NAME>"]` form in here (while we want to deprecate it, good to have that in the list)

Reply:
Also missing docs on `NODEORD`, `PATH`, and `NODELABEL`